### DISCUSSION TITLE
lim_val_batches and val_check_interval behavior

### DISCUSSION THREAD

**Discussion by toskua** [Lightning-AI/pytorch-lightning#8726] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi!

I want to know if validation is performed on the same sub-dataset when I use a combination of the lim_val_batches and val_check_interval flags.

For example, I'm setting lim_val_batches=10 and val_check_interval=1000. Since my training data is huge, validation is called multiple times. Now I want to know if validation is being done on the same 10 batches every time. 

One solution is to simply make my validation set small and set lim_val_batches=1.0. But what if I don't want to do that?
